lib/model/Item.rb

```diff
@@ -1,7 +1,10 @@
 module Shop
   class Item
+    attr_reader :product_id, :quantity
+
     def initialize(product_id, quantity)
-      @product_id, @quantity = product_id, quantity
+      @product_id = product_id
+      @quantity = quantity
```

Comment:
👍 parallel assignment is great solution in some cases, but here you made a step in the right way.